ui/scripts/sharedFunctions.js

```diff
@@ -1208,17 +1208,21 @@ cloudStack.validate = {
 
 cloudStack.preFilter = {
     createTemplate: function(args) {
+        args.$form.find('.form-item[rel=templateSource]').hide();
         if (isAdmin()) {
             args.$form.find('.form-item[rel=isPublic]').css('display', 'inline-block');
             args.$form.find('.form-item[rel=isFeatured]').css('display', 'inline-block');
             args.$form.find('.form-item[rel=isrouting]').css('display', 'inline-block');
+            args.$form.find('.form-item[rel=templatetype]').show();
         } else {
             if (g_userPublicTemplateEnabled == "true") {
                 args.$form.find('.form-item[rel=isPublic]').css('display', 'inline-block');
             } else {
                 args.$form.find('.form-item[rel=isPublic]').hide();
             }
             args.$form.find('.form-item[rel=isFeatured]').hide();
+            args.$form.find('.form-item[rel=templatetype]').hide();
+            args.$form.find(".form-item[rel='activate']").hide();
```

Comment:
I see a `hide()` but not a `show()` for `args.$form.find(".form-item[rel='activate']")`
will this work?

Reply:
Tested, works fine